app/code/Magento/CatalogInventoryGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,7 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+interface ProductInterface {
+    only_x_left_in_stock: Float @doc(description: "Product stock only x left count") @resolver(class: "Magento\\CatalogInventoryGraphQl\\Model\\Resolver\\OnlyXLeftInStockResolver")
+    stock_status: Int @doc(description: "Stock status of the product") @resolver(class: "Magento\\CatalogInventoryGraphQl\\Model\\Resolver\\StockStatusProvider")
```

Comment:
Please consider changing the type of `stock_status` to enum. See example [here](https://github.com/magento/magento2/blob/19dbdde998c136c557b14df1149c0ce48ff7060b/app/code/Magento/CatalogGraphQl/etc/schema.graphqls#L211)